### PR TITLE
Hkobew/UI changes

### PR DESCRIPTION
**Pull request by Hweinstock**
## Solution
New icons: 
image
New help/exit button: 
image
Back button if not first prompt: 
image

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2580,7 +2580,7 @@
                 "command": "aws.saveCurrentLogStreamContent",
                 "title": "%AWS.command.saveCurrentLogStreamContent%",
                 "category": "%AWS.title%",
-                "icon": "$(save-as)",
+                "icon": "$(cloud-download)",
```
Comment (justinmk3) [inline package.json:2583]:
```suggestion
                "icon": "$(save)",
```

File: src/cloudWatchLogs/commands/searchLogGroup.ts [hunk 2]
```diff
@@ -148,15 +149,29 @@ async function logGroupsToArray(logGroups: AsyncIterableIterator<CloudWatchLogs.
...
+    return createInputBox(options)
+    //[createHelpButton(helpUri), createExitButton()]
 }
 
 export function createRegionSubmenu() {
-    return new RegionSubmenu(getLogGroupsFromRegion)
+    return new RegionSubmenu(
+        getLogGroupsFromRegion,
+        { title: 'Select Log Group' },
+        { title: 'Select Region for Log Group' }
```
Comment (justinmk3) [inline src/cloudWatchLogs/commands/searchLogGroup.ts:173]: localize()

File: src/cloudWatchLogs/timeFilterSubmenu.ts [hunk 3]
```diff
@@ -146,10 +149,6 @@ export class TimeFilterSubmenu extends Prompter<TimeFilterResponse> {
         if (Date.parse(startTime) > Date.parse(endTime)) {
             return 'first date must occur before second date'
```
Comment (justinmk3) [inline src/cloudWatchLogs/timeFilterSubmenu.ts:150]: localize()

File: src/cloudWatchLogs/commands/searchLogGroup.ts [hunk 4]
```diff
@@ -179,7 +194,9 @@ export class SearchLogGroupWizard extends Wizard<SearchLogGroupWizardResponse> {
         })
 
         this.form.submenuResponse.bindPrompter(createRegionSubmenu)
-        this.form.filterPattern.bindPrompter(({ submenuResponse }) => createFilterpatternPrompter(submenuResponse!.data))
+        this.form.filterPattern.bindPrompter(({ submenuResponse }) =>
+            createFilterpatternPrompter(submenuResponse!.data, logGroupInfo ? true : false)
```
Comment (JadenSimon) [inline src/cloudWatchLogs/commands/searchLogGroup.ts:198]: This works but is somewhat brittle. Just make sure there are wizard tests asserting that `filterPattern` _always_ shows first if `logGroupInfo` is not provided.

**Review by justinmk3 — APPROVED** [review]

**Review by JadenSimon — APPROVED** [review]

File: src/cloudWatchLogs/commands/searchLogGroup.ts (hunk 4 above)
Comment (Hweinstock) [reply src/cloudWatchLogs/commands/searchLogGroup.ts:198]: I believe there is one on line 123 of `searchLogGroup.test.ts`. Lmk if you had something else in mind.

File: src/cloudWatchLogs/commands/searchLogGroup.ts [hunk 5]
```diff
@@ -152,12 +154,17 @@ async function logGroupsToArray(logGroups: AsyncIterableIterator<CloudWatchLogs.
 export function createFilterpatternPrompter(logGroupName: string, isFirst: boolean): InputBoxPrompter {
     const helpUri =
         'https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html#matching-terms-events'
+    const titleText = localize(
+        'AWS.cwl.searchLogGroup.filterPatternTitle',
+        `Search Log Group ${truncate(logGroupName, -50)}`
+    )
+    const placeHolderText = localize(
+        'AWS.cwl.searchLogGroup.filterPatternPlaceholder',
+        'search pattern (or empty for all events'
```
Comment (justinmk3) [inline src/cloudWatchLogs/commands/searchLogGroup.ts:163]:
```suggestion
        'search pattern (or empty for all events)'
```

File: src/cloudWatchLogs/commands/searchLogGroup.ts [hunk 6]
```diff
@@ -152,12 +154,17 @@ async function logGroupsToArray(logGroups: AsyncIterableIterator<CloudWatchLogs.
 export function createFilterpatternPrompter(logGroupName: string, isFirst: boolean): InputBoxPrompter {
     const helpUri =
         'https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/FilterAndPatternSyntax.html#matching-terms-events'
+    const titleText = localize(
+        'AWS.cwl.searchLogGroup.filterPatternTitle',
+        `Search Log Group ${truncate(logGroupName, -50)}`
+    )
```
Comment (justinmk3) [inline src/cloudWatchLogs/commands/searchLogGroup.ts:160]:
```suggestion
    const titleText = localize(
        'AWS.cwl.searchLogGroup.filterPatternTitle',
        `Search Log Group {0}`,
        truncate(logGroupName, -50)
    )
```

**Review by justinmk3 — APPROVED** [review]

**Pull request merged**